Decred Wallet/Features/Transactions/TransactionTableViewCell.swift

```diff
@@ -0,0 +1,104 @@
+
+import UIKit
+import Dcrlibwallet
+
+class TransactionTableViewCell: UITableViewCell {
+    @IBOutlet weak var txTypeIconImageView: UIImageView!
+    @IBOutlet weak var txAmountOrTicketStatusLabel: UILabel! // holds amount for regular txs and ticket status for staking txs
+    @IBOutlet weak var stakingTxAmountLabel: UILabel! // staking txs only, holds amount for different ticket states
+    @IBOutlet weak var voteRewardLabel: Label! // vote tx only
+    @IBOutlet weak var txDateLabel: UILabel!
+    @IBOutlet weak var daysCounterLabel: UILabel! // voted, revoked and expired tickets only
+    @IBOutlet weak var txStatusIconImageView: UIImageView!
+
+    override class func height() -> CGFloat {
+        return 56
+    }
+
+    func displayInfo(for transaction: Transaction) {
+        let txConfirmations = transaction.confirmations
+        let isConfirmed = Settings.spendUnconfirmed || txConfirmations > 1
+
+        let txDate = Date(timeIntervalSince1970: TimeInterval(transaction.timestamp))
+        let ageInDays = txDate.daysFromNow
+        var txDateString: String
+
+        if ageInDays == 0 {
+            txDateString = LocalizedStrings.today
+        } else if ageInDays == -1 {
+            txDateString = LocalizedStrings.yesterday
+        } else {
+            let txDateIsInCurrentYear = Calendar.current.isDate(txDate, equalTo: Date(), toGranularity: .year)
+            txDateString = txDate.toString(format: txDateIsInCurrentYear ? "MMM dd" : "MMM dd, YYYY")
+        }
+
+        self.txDateLabel.text = isConfirmed ? txDateString : LocalizedStrings.pending
+        self.txDateLabel.textColor = isConfirmed ? UIColor.appColors.bluishGray : UIColor.appColors.lightBluishGray
+        self.txStatusIconImageView.image = isConfirmed ? UIImage(named: "ic_confirmed") : UIImage(named: "ic_pending")
+
+        self.stakingTxAmountLabel.isHidden = transaction.type == DcrlibwalletTxTypeRegular
+        self.daysCounterLabel.isHidden = transaction.type == DcrlibwalletTxTypeRegular
+        self.voteRewardLabel.isHidden = true //TODO: set value
+
+        if transaction.type == DcrlibwalletTxTypeRegular {
+            self.displayRegularTxInfo(transaction)
+        } else if transaction.type == DcrlibwalletTxTypeVote {
+            self.displayVoteTxInfo(transaction)
+        } else if transaction.type == DcrlibwalletTxTypeTicketPurchase {
+            self.displayTicketPurchaseInfo(transaction)
+        }
+    }
+    
+    func displayRegularTxInfo(_ transaction: Transaction) {
+        let amountString = Utils.getAttributedString(str: transaction.dcrAmount.round(8).description, siz: 13.0, TexthexColor: UIColor.appColors.darkBlue)
+
+        if transaction.direction == DcrlibwalletTxDirectionSent {
+            let attributedString = NSMutableAttributedString(string:"-")
+            attributedString.append(amountString)
+            self.txAmountOrTicketStatusLabel.attributedText = attributedString
```

Comment:
This can be a 1-liner. Too long?

```suggestion
            self.txAmountOrTicketStatusLabel.attributedText = NSMutableAttributedString(string:"-").append(amountString)
```

Reply:
NSMutableAttributedString(string:"-").append(amountString) returns Void